osgar/drivers/system_monitor.py

```diff
@@ -0,0 +1,42 @@
+""""
+    Records system information like CPU and RAM utilization or device temperatures.
+"""
+
+import psutil
+# https://psutil.readthedocs.io/en/latest/#system-related-functions
+from threading import Thread
+
+class SystemMonitor:
+    def __init__(self, config, bus):
+        self.input_thread = Thread(target=self.run_input, daemon=True)
+        self.bus = bus
+        bus.register('cpu', 'ram', 'temp')
+        self.sleep = config.get('sleep', 1)
+
+    def start(self):
+        self.input_thread.start()
+
+    def join(self, timeout=None):
+        self.input_thread.join(timeout=timeout)
+
+    def run_input(self):
+        while self.bus.is_alive():
+            # current system-wide CPU utilization as a percentage
+            cpu = psutil.cpu_percent(percpu=True)
+            if cpu:
```

Comment:
why is here this if? What could be the answer - `None`? If yes, then I would prefer to change the statements to `if cpu is not None:`. Thanks